content/download/releaseNotes/releaseNotes8.adoc

```diff
@@ -17,8 +17,17 @@ and makes a few minor backwards incompatible changes.
 Just follow the instructions in link:../upgradeRecipe/upgradeRecipe8.html[the upgrade recipe].
 
 As usual, Red Hat offers link:../../product/services.html[enterprise support and consulting] on OptaPlanner 8,
-with the _Red Hat build of OptaPlanner_ in an upcoming release of Red Hat Decision Manager 7
-in the first half of 2021.
+with the _Red Hat build of OptaPlanner 8_ available and fully supported
+in the latest release of Red Hat Decision Manager 7.
+
+== New and noteworthy: Engine 8.10.0.Final
+
+=== Java 11 or later required
+
+Together with the Quarkus and Kogito projects, we have raised the minimum required version of the Java Platform to 11.
+OpenJDK is an open source Java implementation and is https://adoptopenjdk.net/[free to download].
```

Comment:
Officially, we already stated that OptaPlanner 8.0 required at least Java 11.
This messaging implies that we're doing it between version 8.9 and 8.10, which looks bad: we can't change the java version between minor versions.

Reply:
My proposal: in the release notes, I wouldn't mention anything, because that is the good news show.
In the upgrade recipe (the bad news show), "From 7.48+ to 8.0.0.Final" already has "Java 11 or higher required",
but "From 8.9.0.Final to 8.10.0.Final" can state that the class files are now target=11 too while referring to the fact that we require Java 11 or higher as of version 8.0 already. Add the true cause of this (Quarkus 1.x used to require java 8 support) is probably a good idea, so people understand we used target=8 temporarily for being practical.